akka-stream/src/main/scala/akka/stream/impl/fusing/ActorGraphInterpreter.scala

```diff
@@ -505,7 +514,7 @@ private[stream] final class GraphInterpreterShell(
       _isTerminated = true
```

Comment:
yes, we are already aborting and re-aborting if abort failed makes no sense, we just give up.